src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,24 +1,102 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xmlns.jcp.org/xml/ns/javaee" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd" id="WebApp_ID" version="4.0">
-  <display-name>workshop-servlet-khadri-mart</display-name>
-  <welcome-file-list>
-    <welcome-file>main.html</welcome-file>
-  </welcome-file-list>
-  <servlet>
-    <servlet-name>groadd</servlet-name>
-    <servlet-class>com.khadri.mart.grocessary.servlet.AddGrossaryServlet</servlet-class>
-  </servlet>
-  <servlet-mapping>
-   <servlet-name>groadd</servlet-name>
-   <url-pattern>/groadd</url-pattern>
-  </servlet-mapping>
-  
-   <servlet>
-    <servlet-name>gromodify</servlet-name>
-    <servlet-class>com.khadri.mart.grocessary.servlet.ModifyDynaGrossaryServlet</servlet-class>
-  </servlet>
-  <servlet-mapping>
-   <servlet-name>gromodify</servlet-name>
-   <url-pattern>/gromodify</url-pattern>
-  </servlet-mapping>
-</web-app>
\ No newline at end of file
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
+	id="WebApp_ID" version="4.0">
+	<display-name>workshop-servlet-khadri-mart</display-name>
+	<welcome-file-list>
+		<welcome-file>main.jsp</welcome-file>
+	</welcome-file-list>
+	<servlet>
+		<servlet-name>groadd</servlet-name>
+		<servlet-class>com.khadri.mart.grosary.servlet.AddGrosaryServlet</servlet-class>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>groadd</servlet-name>
+		<url-pattern>/groadd</url-pattern>
+	</servlet-mapping>
+	<servlet>
+		<servlet-name>jspadd</servlet-name>
+		<jsp-file>/groAdd.jsp</jsp-file>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>jspadd</servlet-name>
+		<url-pattern>/grosary_add</url-pattern>
+	</servlet-mapping>
+	<servlet>
+		<servlet-name>gromodify</servlet-name>
+		<servlet-class>com.khadri.mart.grosary.servlet.ModifyGrosaryServlet</servlet-class>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>gromodify</servlet-name>
+		<url-pattern>/gromodify</url-pattern>
+	</servlet-mapping>
+	<servlet>
+		<servlet-name>modifyJsp</servlet-name>
+		<jsp-file>/groModify.jsp</jsp-file>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>modifyJsp</servlet-name>
+		<url-pattern>/grosary_modify</url-pattern>
+	</servlet-mapping>
+	<servlet>
+		<servlet-name>modifypageJsp</servlet-name>
+		<jsp-file>/grosary_modify_page.jsp</jsp-file>
+	</servlet>
+	<servlet-mapping>
+		<servlet-name>modifypageJsp</servlet-name>
+		<url-pattern>/grosary_modify_page</url-pattern>
+	</servlet-mapping>
+	<servlet>
+		<servlet-name>jspview</servlet-name>
```

Comment:
change jspview to viewjsp  or groviewjsp

Reply:
ok